### DISCUSSION TITLE
Layer Shell support

### DISCUSSION THREAD

**Discussion by lnxstar** [CuarzoSoftware/Louvre#4] [Ideas]
Hello, I'm curious about the support for the [layer shell](https://wayland.app/protocols/wlr-layer-shell-unstable-v1). I've reviewed the documentation, and it appears that support is currently unavailable. Are there any plans to include it in the future? There are several exciting applications that make use of it, and having support for the layer shell would be a valuable addition. Thank you!

**Comment by ehopperdietzel** [comment]
Certainly! It appears to be quite useful, though I'm undecided on whether to enable it by default (meaning [adding its global](https://cuarzosoftware.github.io/Louvre/class_louvre_1_1_l_compositor.html#a1e03e3983ef2f3495537eb4ff0fc9996) during compositor initialization). I find it somewhat "invasive", but I plan to experiment with adding it when I have more time.

**Reply by lnxstar** [reply]
Great! Looking forward to that, and yes, maybe keeping it optional would be a good decision.

**Comment by lnxstar** [comment]
I've just realized that it is now supported, and I must say it works perfectly. Great job!
